osi_environment.proto

```diff
@@ -324,4 +336,163 @@ message EnvironmentalConditions
+            //
+            CLOUD_LAYER_LEVEL_1 = 3;
+
+            // 2/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_2 = 4;
+
+            // 3/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_3 = 5;
+
+            // 4/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_4 = 6;
+
+            // 5/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_5 = 7;
+
+            // 6/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_6 = 8;
+
+            // 7/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_7 = 9;
+
+            // 8/8 of the sky is covered with clouds.
+            //
+            CLOUD_LAYER_LEVEL_8 = 10;
+
+            // Sky obscured, describes situations where the sky is not perceivable, e.g. in dense fog.
+            //
+            CLOUD_LAYER_LEVEL_SKY_OBSCURED = 11;
+        }
+    }
+
+    //
+    // \brief Defines wind properties.
+    //
+    message Wind
+    {
+        // The origin direction of the wind (not the target direction) in the ground/xy-plane of the \c
+        // world coordinate system. Corresponds to the heading/yaw angle, counted counterclockwise. \c
+        // 0 pointing north. If north not explicitly defined via proj4 reference in ground truth, \c
+        // then north is pointing in y-axis direction. The default orientation (x, y, z) is easting, northing, up [1].
+        //
+        // Unit: rad
+        //
+        // The preferred angular range is [0, 2pi].
+        //
+        // \note The direction is the origin, not the target direction.
+        //
+        // \par References:
+        // [1] https://proj.org/usage/projections.html#:~:text=E%20%20%20%200dN%200.000-,Axis%20orientation,%C2%B6,-Starting%20in%20PROJ
+        //
+        optional double origin_direction = 1;
+
+        // The wind speed.
+        //
+        // Unit: m/s
+        //
+        // Range [0...inf[
+        //
+        optional double speed = 2;
+    }
+
+    //
+    // \brief Specification of sun properties.
+    //
+    message Sun
+    {
+        // Azimuth of the sun, counted counterclockwise.
+        // 0 pointing north. If north not explicitly defined via proj4 reference in ground truth, \c
+        // then north is pointing in y-axis direction. The default orientation (x, y, z) is “easting, northing, up” [1].
+        //
+        // \note 0: north; +pi/2: west; pi: south, 3/2 pi: east.
+        //
+        // Unit: rad
+        //
+        // The preferred angular range is [0, 2pi].
+        //
+        // \par References:
+        // [1] https://proj.org/usage/projections.html#:~:text=E%20%20%20%200dN%200.000-,Axis%20orientation,%C2%B6,-Starting%20in%20PROJ
+        //
+        optional double azimuth = 1;
+
+        // Solar elevation angle. Counted in positive direction around x-axis.
```

Comment:
Maybe misleading description with the x-axis, since the elevation angle does not refer to it.
Maybe "Counted positive for an elevation above the xy-plane"?